core/Url.php

```diff
@@ -669,6 +664,24 @@ public static function isHostInUrls($host, $urls)
             }
         }
 
-        return in_array($host, self::$alwaysTrustedHosts);
+        return in_array($host, self::getAlwaysTrustedHosts());
+    }
+
+    /**
+     * List of hosts that are never checked for validity.
+     *
+     * @return array
+     */
+    private static function getAlwaysTrustedHosts()
```

Comment:
the class already has a mix of protected and private methods, I put private because we're not supposed to extend this class. I think it does not matter either way?